### PR TITLE
[HOPSWORKS-1001] Experiments service refactor certificates cleanup

### PR DESCRIPTION
**Pull request by robzor92**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [x] Passes the tests
- [x] HOPSWORKS JIRA issue has been opened for this PR
- [x] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing/pull/348

**Comment by robzor92** [comment]
@tkakantousis please review, I moved two error codes from ServiceException to TensorBoardException

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/tensorflow/config/TensorBoardProcessMgr.java [hunk 1]
```diff
@@ -303,43 +294,64 @@ public int killTensorBoard(TensorBoard tb) {
...
+            .build();
+
+    LOGGER.log(Level.FINE, processDescriptor.toString());
+    try {
+      ProcessResult processResult = osProcessExecutor.execute(processDescriptor);
+      if (!processResult.processExited()) {
+        LOGGER.log(Level.SEVERE, "Failed to remove TensorBoard directory, process time-out");
       }
+    } catch (IOException ex) {
+      LOGGER.log(Level.SEVERE, "Could not delete TensorBoard directory: " + tensorBoardDirectoryPath);
```
Comment (tkakantousis) [inline hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/tensorflow/config/TensorBoardProcessMgr.java]: @robzor92 remove redundant logger

File: hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java [hunk 2]
```diff
@@ -1346,4 +1343,46 @@ public int getRange() {
     }
   }
 
+  /**
+   * Airflow specific error codes
```
Comment (tkakantousis) [inline hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java]: @robzor92 copy paste leftover

File: hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java [hunk 3]
```diff
@@ -1346,4 +1343,46 @@ public int getRange() {
...
+
+    TENSORBOARD_CLEANUP_ERROR(1, "Failed when deleting a running TensorBoard", Response.Status.INTERNAL_SERVER_ERROR),
+    TENSORBOARD_START_ERROR(2, "Failed to start TensorBoard", Response.Status.INTERNAL_SERVER_ERROR),
+    TENSORBOARD_FETCH_ERROR(3, "Error while fetching TensorBoard from database",
+            Response.Status.INTERNAL_SERVER_ERROR);
+
+    private Integer code;
+    private String message;
+    private Response.StatusType respStatus;
+    private final int range = 300000;
```
Comment (tkakantousis) [inline hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java]: @robzor92 should be `28000`, not `300000`

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/tensorflow/config/TensorBoardProcessMgr.java (hunk 1 above)
Comment (robzor92) [reply hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/tensorflow/config/TensorBoardProcessMgr.java]: Fixed

File: hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java (hunk 2 above)
Comment (robzor92) [reply hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java]: Fixed

File: hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java (hunk 3 above)
Comment (robzor92) [reply hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java]: Fixed

**Pull request merged**